tests/unit/test_extractor_query.py

```
@@ -0,0 +1,707 @@
+	{'type': 'target', 'field': 'name', 'condition': "target.type == 'host'"},           # equality
+	{'type': 'target', 'field': 'name', 'condition': "item.type == 'url'"},              # equality (item.)
+	{'type': 'tag', 'field': 'name', 'condition': "item.name == 'email_address'"},       # equality (item.)
+	{'type': 'url', 'field': 'url', 'condition': "item.stored_response_path != ''"},     # inequality str
+	{'type': 'url', 'field': 'url', 'condition': 'item.status_code != 0'},               # inequality int
+	{'type': 'tag', 'field': 'name', 'condition': "item.name in ['sqli']"},              # in [...]
+	# regex forms (unified ~=, case-insensitive by default) that replaced startswith/.lower()/in:
+	{'type': 'url', 'field': 'url', 'condition': 'item._source ~= ^httpx',
+	 'old': "item._source.startswith('httpx')"},
+	{'type': 'url', 'field': 'url',
+	 'condition': 'item._source ~= ^urlparser or item._source ~= ^arjun or item._source ~= ^x8',
+	 'old': "item._source.startswith('urlparser') or item._source.startswith('arjun') "
```

Flake8 (7.3.0)

[error] indentation contains mixed spaces and tabs

(E101)